frontend/javascripts/navbar.tsx

```diff
@@ -172,7 +172,7 @@ function UserInitials({
             height: 12,
             lineHeight: "12px",
             fontSize: 12,
-            color: "#75df4a",
+            color: "#59F8E8",
```

Comment:
could you put the style into a css class into `main.less`? Then, you can use `@blue-zircon` (compare with the `.ant-avatar` class).